onnx/test/shape_inference_test.py

```diff
@@ -232,6 +232,24 @@ def test_reshape_static_shape_inferred(self):  # type: () -> None
             initializer=[make_tensor('shape', TensorProto.INT64, (3,), (0, 3, -1))])
         self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.UINT8, (2, 3, 4))])
 
+    def test_reshape_static_shape_zero(self):  # type: () -> None
+        graph = self._make_graph(
+            [('x', TensorProto.UINT8, (1, 0, 0)),
```

Comment:
Can we make the shape of 'x' (1,1,1) ? Otherwise, this example is not a valid example, since the input and output don't have same size. Better to avoid invalid examples in our test-suite.

Reply:
In the future, we could technically extend the shape-inference to reject such a test-case.

Reply:
Thx for catching, updated.